lib/spack/spack/util/environment.py

```diff
@@ -25,23 +25,10 @@
 import os
 
 system_paths = ['/', '/usr', '/usr/local']
-suffixes = ['lib', 'lib64', 'include']
-system_dirs = [os.path.join(p, s) for s in suffixes for p in system_paths] + \
-    system_paths
-system_bins = [os.path.join(p, 'bin') for p in system_paths]
 
 
 def filter_system_paths(paths):
-    return [p for p in paths if p not in system_dirs]
```

Comment:
This change doesn't break anything. Previously, things like `/usr` and `/usr/lib` would both get filtered. Now, only `/usr` gets filtered. This function is only use once on the 4 lines you see above, all of which use a prefix, not a prefix.lib or prefix.include. So I could tell it to also filter things like `/usr/include`, but it wouldn't make a difference.